### PR TITLE
Add Theia IDE

### LINKED ISSUE #468: Add "Theia IDE"

### PR DESCRIPTION
**Pull request by JonasHelming**
fixed #468 

- [x] Fix margins
- [x] Refine feature videos
- [x] Fix all references to Blueprint on remaining website including documentation
- [x] Complete VS Code extension gallery with links and more entries
- [x] Integrate logo well, e.g. on the top right corner

this this as a follow up: https://github.com/eclipse-theia/theia-website/issues/485

### CODE REVIEW

**Comment by emilhammarstedtst** [comment]
Dark mode!?
Screenshot from 2023-11-07 16-20-15

File: src/components/index/TheiaIDEDownloads.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                 </div>
+                
+                 <center>
+                 You can also <a href="https://try.theia-cloud.io/">try the latest version of the Theia IDE online</a>. The online test version is limited to 30 minutes per session and hosted via <a href="https://github.com/eclipsesource/theia-cloud">Theia.cloud</a>.
+                 </center>
+                 <br></br>
+            </div>
+            <div className="row">
+                <h1>Feature Requests and Bug Reports</h1>
+                Help us make the Theia IDE even better, by sharing your experience and suggestions <a href="https://github.com/eclipse-theia/theia/discussions">here</a>.
```
Comment (planger) [inline src/components/index/TheiaIDEDownloads.js:107]:
```suggestion
                Help us make the Theia IDE even better, by <a href="https://github.com/eclipse-theia/theia/discussions">sharing your experience and suggestions</a>.
```

File: src/components/index/TheiaIDEDownloads.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                The features found in the Eclipse Theia IDE are based on Eclipse Theia and the available extensions/plugins. For more details please see the <a href="https://github.com/eclipse-theia/theia">Eclipse Theia GitHub Repository</a>. The Eclipse Theia IDE only packages existing Eclipse Theia functionality and installers. If you believe there is a mistake in packaging, something needs to be added to the packaging or the installers do not work properly, please <a href="https://github.com/eclipse-theia/theia-blueprint/issues/new/choose">open an issue on GitHub</a> to let us know.
+                <h1>Support</h1>
+                Need help with Theia? To get support by the community go to the <a href="https://community.theia-ide.org/">Discourse Theia forum</a> or the <a href="https://github.com/eclipse-theia/theia/discussions">Discussions at GitHub</a>. To get professional support for Theia see the <a href="http://localhost:8000/support/">support page</a>.
```
Comment (planger) [inline src/components/index/TheiaIDEDownloads.js:110]:
```suggestion
                Need help with Theia? To get support by the community go to the <a href="https://community.theia-ide.org/">Discourse Theia forum</a> or the <a href="https://github.com/eclipse-theia/theia/discussions">Discussions at GitHub</a>. To get professional support for Theia see the <a href="/support/">support page</a>.
```

**Review by planger — COMMENTED** [review]
Very nice, thank you very much! I found just two minors which I marked inline.

**Comment by JonasHelming** [comment]
@planger I committed your suggestions, thanks, could you apporve?

**Review by planger — APPROVED** [review]
Thanks, looks good to me!

In the preview though, the padding of the buttons in the main banner on top seems to be missing now:
image

Except for that, I think this is all perfect! Thank you!

**Comment by JonasHelming** [comment]
Hmm, the padding works for me locally, but also here in the preview :-/

**Pull request merged**